strr-web/components/bcros/form-section/property/Form.vue

```diff
@@ -69,20 +88,31 @@ const { isComplete } = defineProps<{
 const addressNotInBC = ref(false)
 
 const {
-  address: canadaPostAddress,
+  activeAddressField,
+  addressWithStreetAttributes: canadaPostAddress,
   enableAddressComplete
-} = useCanadaPostAddress()
+} = useCanadaPostAddress(true)
+
+const getActiveAddressState = () => {
+  if (activeAddressField.value === 'propertyAddressStreetNumber' ||
+      activeAddressField.value === 'propertyAddressStreetName') {
+    return formState.propertyDetails
+  }
+  return null
+}
 
 watch(canadaPostAddress, (newAddress) => {
-  if (newAddress) {
+  const activeAddressState = getActiveAddressState()
```

Comment:
Add a type for safety, either to the function or here.

Reply:
Fixed